egui/src/style.rs

```diff
@@ -785,7 +819,8 @@ impl Default for Widgets {
 
 // ----------------------------------------------------------------------------
 
-use crate::{widgets::*, Ui};
+use crate::widgets::*;
```

Comment:
why did you split this up?

Reply:
rustfmt with rust analyzer. idk what setting caused this 😤

Reply:
ah ok. it's the settings in egui's `.vscode` folder then. nevermind! xd